server/graphql/v1/mutations/collectives.js

```diff
@@ -634,7 +634,19 @@ export async function archiveCollective(_, args, req) {
     throw new Error('Cannot archive collective with balance > 0');
   }
 
-  return collective.update({ isActive: false, deactivatedAt: Date.now() });
+  const membership = await models.Member.findOne({
+    where: {
+      CollectiveId: collective.id,
+      MemberCollectiveId: collective.HostCollectiveId,
+      role: roles.HOST,
+    },
+  });
+
+  if (membership) {
+    membership.destroy();
+  }
```

Comment:
Thank you for adding that. I think we need to also add `HostCollectiveId: null`in `collective.update`.

Reply:
Yes we do, I just added it.